tests/test_flow/test_persistence_fuzz.py

```diff
@@ -221,14 +255,37 @@ def run(self, n_iterations):
                     returned_value = self.model.build_flow().get(affected_name)
                     expected_value = self.model.expected_entity_value(
                         affected_name)
-                    if make_func_change:
+
+                    # Returned value will be different from expected value if we made
```

Comment:
This first sentence seems to be just restating the first part of the if statement. It seems like the key idea here is is that under certain circumstances, Bionic will miss the fact that a change occurred.

Also, the comment immediately above is no longer always correct, right?

Reply:
It's not. I guess the comment is pretty unclear. So the returned value from the flow won't be recomputed because the provenance will not change in the if conditional. Hence, the value from the flow won't be the same as expected value which is the actual recomputed value.

But if don't make a functional change, then the values will still be the same. In similar fashion, if we make a functional change and the entity is nondeterministic, it will be recomputed and hence return the updated value. Let's take this discussion to slack.